Tests/Integration/config/messenger.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+    messenger:
+        transports:
+            async: 'in-memory://'
+        routing:
+            'Presta\SitemapBundle\Messenger\DumpSitemapMessage': async
+
+services:
+    Presta\SitemapBundle\Tests\Integration\Controller\MessengerController:
+        tags: ['controller.service_arguments']
+#    Presta\SitemapBundle\Messenger\DumpSitemapMessageHandler:
```

Comment:
The previous comment was refering to this particular service, you may want to remove these lines now.